### PR TITLE
Spelling tests

### PR DESCRIPTION
**Pull request by jsoref**
This is the remainder of tests noted in #50754, it's _around_ 50 files and < 100 changes, which is hopefully digestible.

I'm not going to send any other PRs at this time as I'd like some input on how to slice the rest of the bits.

### CODE REVIEW

File: tests/lib/mirrors/mirrors_reader.dart [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICESNE file.
+// BSD-style license that can be found in the LICENSE file.
```
Comment (jsoref) [inline tests/lib/mirrors/mirrors_reader.dart:3]: License changes historically spook lawyers, but this shouldn't be a problem...

File: tests/standalone/io/link_test.dart [hunk 2]
```diff
@@ -239,7 +239,7 @@ testRenameSync() {
 
 void testLinkErrorSync() {
   Expect.throws(
-      () => new Link('some-dir-that-doent exist/some link file/bla/fisk')
+      () => new Link('some-dir-that-does-not exist/some link file/bla/fisk')
```
Comment (jsoref) [inline tests/standalone/io/link_test.dart:242]: I personally prefer `nonexistent`, but this seems like the simplest change matching the spirit of the test,

Comment (eernstg) [reply tests/standalone/io/link_test.dart:242]: Should be fine.

File: tests/lib/mirrors/mirrors_reader.dart (hunk 1 above)
Comment (eernstg) [reply tests/lib/mirrors/mirrors_reader.dart:3]: Indeed.

**Comment by eernstg** [comment]
Got this one: https://dart-review.googlesource.com/c/sdk/+/276360. Currently checking that the changes do not cause any new test failures. Several identifiers were modified, but they look like every modified name has been changed consistently, and the name is declared in the same library (so we aren't breaking code because it refers to a misspelled imported name).

**Pull request closed without merging**